src/SprykerEco/Zed/AfterPay/Business/Order/Saver.php

```diff
@@ -140,4 +140,18 @@ protected function getIdChannel(string $paymentMethod): string
     {
         return $this->config->getPaymentChannelId($paymentMethod);
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return string|null
+     */
+    protected function getIdCheckout(QuoteTransfer $quoteTransfer): ?string
+    {
+        if ($quoteTransfer->getAfterPayAvailablePaymentMethods()) {
```

Comment:
it would be nice to change it to: $quoteTransfer->getAfterPayAvailablePaymentMethods() !== null